qutebrowser/config/configtypes.py

```diff
@@ -1760,11 +1760,14 @@ def to_py(self, value: _StrUnset) -> _StrUnsetNone:
         elif not value:
             return None
 
-        if not ('{}' in value or '{0}' in value):
+        if not re.search('{(|0|quoted)}', value):
             raise configexc.ValidationError(value, "must contain \"{}\"")
 
         try:
-            value.format("")
+            format_keys = {
+                'quoted': "",
+            }
+            value.format("", **format_keys)
         except (KeyError, IndexError):
             raise configexc.ValidationError(
                 value, "may not contain {...} (use {{ and }} for literal {/})")
```

Comment:
I can't comment on the proper line - but I'm pretty sure the `QUrl` check below would fail with a value only using `{quoted}` but not `{}`, no?

Reply:
I'm not sure whether `QUrl` check still makes sense if we allow `{}`, `{quoted}` and `{unquoted}`.